proto_parser/src/proto_parser.cc

```diff
@@ -78,8 +78,12 @@ std::unique_ptr<ProtoValue> ProtoParser::CreateMessage(
     const google::protobuf::Message& message, 
     const google::protobuf::TextFormat::ParseInfoTree& tree, int indent_count,
     int field_loc, absl::string_view name){
-  // TODO(alexanderlin): Implement.
-  return nullptr;
+  std::unique_ptr<ProtoValue> message_val = 
+      absl::make_unique<MessageValue>(std::string(name), indent_count, 
```

Comment:
Do you need string(name) here? Cannot pass absl::string_view directly?

Reply:
This was due to in old implementation the constructor took a const string& instead of string_view, this is changed now.